framework/web/helpers/CHtml.php

```diff
@@ -2220,7 +2224,11 @@ public static function errorSummary($model,$header=null,$footer=null,$htmlOption
 				foreach($errors as $error)
 				{
 					if($error!='')
-						$content.= '<li>'.$error."</li>\n";
+					{
+                        if (!isset($htmlOptions['ignoreHtmlEncode']) || !$htmlOptions['ignoreHtmlEncode'])
```

Comment:
Let's be consistent in naming: https://github.com/yiisoft/yii/blob/master/framework/web/helpers/CHtml.php#L153

Reply:
Sorry, have not seen it. Few moments and I will fix it with tabs.